releases/release-5.4.1.md

```diff
@@ -0,0 +1,175 @@
+---
+title: TiDB 5.4.1 Release Notes
+---
+
+# TiDB 5.4.1 Release Notes
+
+发版日期：2022 年 5 月 13 日
+
+TiDB 版本：5.4.1
+
+## 兼容性更改
+
++ TiDB
+
+    - (dup: release-6.0.0-dmr.md > Bug 修复> TiDB)- 修复 date_format 对 `'\n'` 的处理与 MySQL 不兼容的问题 [#32232](https://github.com/pingcap/tidb/issues/32232)
```

Comment:
This is a bugfix, I think we can remove it.

Reply:
@qiancai Maybe we can move it to the bug fix part?
```suggestion
    - (dup: release-6.0.0-dmr.md > Bug 修复> TiDB)- 修复 date_format 对 `'\n'` 的处理与 MySQL 不兼容的问题 [#32232](https://github.com/pingcap/tidb/issues/32232)
```

Reply:
LGTM